### DISCUSSION TITLE
Enable to get data for REST Services

### DISCUSSION THREAD

**Discussion by nuhaildam** [Mair/esp32-course#95] [Course Qeustions] [answered]
I'm getting a following error when requesting data from "https://quotes.rest/qod
Code Source: [esp32-course](https://github.com/Mair/esp32-course)/[_14_Rest_Client](https://github.com/Mair/esp32-course/tree/master/_14_Rest_Client)/[v4.3](https://github.com/Mair/esp32-course/tree/master/_14_Rest_Client/v4.3)/_1_get_data/
![rest](https://user-images.githubusercontent.com/104188092/165719782-00075003-c229-4bb7-9766-082c4c030ce1.png)

**Comment by Mair** [comment, marked as answer]
for this video use http://quotes.rest/qod (http not https) you will learn about secure connections is the next few videos

**Reply by nuhaildam** [reply]
When I use the above http address it gives the following error
![rest1](https://user-images.githubusercontent.com/104188092/165895309-2392f359-549f-4383-b87c-358ac18d8cbe.png)
But when I use POSTMAN or THUNDER CLIENT there does not seems to have any issue with [quote.rest ](http://quotes.rest/qod ).
All the other REST API's are working fine except API used in the example. What must be the problem?

**Reply by Mair** [reply]
It looks like it may be currently down.
maybe try this one http://www.thunderclient.com/welcome

**Comment by nuhaildam** [comment]
(SOLVED). The problem was with the REST service which I was using.
